bigtable_input.py

```diff
@@ -108,7 +114,7 @@ def make_single_array(ds, batch_size=8*1024):
                 batches.append(sess.run(get_next))
         except tf.errors.OutOfRangeError:
             pass
-    return np.concatenate(batches)
+    return np.concatenate(batches) if batches else []
```

Comment:
For consistency, return an empty np array of the correct dimension?

Reply:
This turned out to be an interesting homework problem! Fixed by limiting the function to require a dataset that produces a one-dimensional array. (It wasn't as general a debugging function as it looked like, because of the behavior of `np.concatenate`).